src/Forms/FormMessage.php

```diff
@@ -92,6 +91,57 @@ public function setMessage(
         return $this;
     }
 
+    /**
+     * Appends a message to the existing message if the types match, overrides if not
+     *
+     * @param string $message Message string
+     * @param string $messageType Message type
+     * @param string $messageCast
+     * @param bool $force Whether or not to override the existing message if appending is invalid
+     * @throws InvalidArgumentException
+     * @return $this
+     */
+    public function appendMessage(
+        string $message,
+        string $messageType = ValidationResult::TYPE_ERROR,
+        string $messageCast = ValidationResult::CAST_TEXT,
+        bool $force = false,
+    ): static {
+        if (empty($this->message)) {
+            return $this->setMessage($message, $messageType, $messageCast);
+        }
+        $canBeMerged = ($messageType === $this->getMessageType() && $messageCast === $this->getMessageCast());
+
+        if (!$canBeMerged && $force === false) {
```

Comment:
```suggestion
        if (!$canBeMerged && !$force) {
```
These are both bool - so either they both get `=== false` treatment or they both get `!` treatment. Having them differ makes it look like one of them can be something other than bool, or for whatever reason needs to be treated specially.

Reply:
Hah yeah not sure how that ended up like that 😅